docs/manual/user_guide/siteonlyapi_tutorial.ipynb

```diff
@@ -6,7 +6,7 @@
    "source": [
     "# Diasaggregate your Home/Building Mains Meter Data\n",
     "\n",
-    "### This notebook demonstrates the use of siteonlyapi - a new NILMTK interface which is a modification of NILMTK's ExperimentAPI. It allows NILMTK users to get their home/buildings energy demands for different potential appliances.\n",
+    "### This notebook demonstrates the use of siteonlyapi - a new flag option within the NILMTK Experiment API interface which allows the user to run disaggregation on only site data without sub-meter information as the testing stage. This of course means that there is no possibility to asses the algorithm's accuracy in these tests. It allows NILMTK users to get their home/buildings energy demands for different potential appliances.\n",
```

Comment:
I agree the current description lacks clarity. I would mention in the explanation the word "unsupervised". What do you think of the following beginning?

> This notebook demonstrates the site_only option of the NILMTK Experiment API. This flag allows the user to run unsupervised disaggregation on site data only, without sub-meter information at the testing stage. [...]

I am happy with the rest of your text.